uiowahjmjohnsonselt2024/projectdirectory-selt_2024_team_009
# Pull request app works with refreshes

## What does this PR do?
This PR fixes the issue where obstacles were not being placed correctly on the game grid. Now, obstacles are randomly placed in five cells that do not already contain treasures. Additionally, the app works correctly upon refreshing the page.



## Testing
1. Start a new game and verify that the grid initializes without errors.
2. Ensure that treasures are placed in random cells as expected.
3. Confirm that obstacles are placed in five random cells where treasures are not present.
4. Refresh the page and confirm that the app still works as expected, with no issues in displaying the grid, treasures, or obstacles.

## Checklist

## Extra information (if needed)
- The app functions as expected even after refreshing the page, ensuring obstacles and treasures remain correctly displayed.
